tests/reset_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors
+ *
+ */
+
+package tests_test
```

Comment:
Do you have a suggestion on  what subpackage the reset test suite belongs in?

Similar subresource tests such as pause, soft-reboot, and even the subresource_api_test.go file are in this top level package which is consistent with where i placed the reset test.

Reply:
I think that this may belong better under `compute`.

Reply:
Suggestion: 
Let's move it under compute where we create a lifecycle package. Move this test to reset.go file and in follow up we can break the huge `tests/vmi_lifecycle_test.go` in the same manner. We also found that there are more lifecycle tests but not under the `tests/vmi_lifecycle_test.go` so we would move them under this package.

It should be easy for us to find tests but for now it is the opposite imho.

Reply:
this feels like we're dipping into refactoring territory that goes beyond the scope of this PR.

The current reset test file follows the same convention as the reboot test file (see `tests/softreboot_test.go`). I'd suggest sticking with the existing convention and refactoring the location these tests belong in a follow up is desired. Otherwise moving only reset to compute makes the situation more fragmented for the similar subresource test files.

Reply:
Currently, we have
```
$ git grep tests_test|wc -l
43
```
and this PR makes it a little bit worse. I am not an approver here so I can only advise, but I would prefer to avoid this worsening, even by introducing a short-lived fragmentation, to be solved by a quick follow-up.

Reply:
I agree with @dankenigsberg. Work to address this issue was merged recently. Can you follow the new conventions please?

See https://github.com/kubevirt/kubevirt/pull/12865

Reply:
Actually, with https://github.com/kubevirt/kubevirt/pull/12865 softreboot and others were already splitted between `vmi_lifecycle` and `vm_lifecycle`, so I think that @xpivarc's suggestion makes sense.

Reply:
#12865 is not completely enough but we are getting somewhere....

Reply:
>Can you follow the new conventions please?

Oh, nice! yeah no problem, done.